internal/blueprint/filesystem_customizations.go

```diff
@@ -9,7 +9,8 @@ import (
 
 type FilesystemCustomization struct {
 	Mountpoint string `json:"mountpoint,omitempty" toml:"mountpoint,omitempty"`
-	MinSize    uint64 `json:"minsize,omitempty" toml:"size,omitempty"`
+	MinSize    uint64 `json:"minsize,omitempty" toml:"minsize,omitempty"`
+	Size       uint64 `toml:"size,omitempty"`
```

Comment:
I'd add a comment to Size making it clear it should never be used outside UnmarshalTOML because it only there as an alias for MinSize. I can see someone (totally not me!) looking at the struct in 6 months and saying 'obviously Size is what I want to use' :)

Reply:
I just realized (and successfully tested), that thanks to the poor implementation of custom unmarshalers in the TOML library that we use, we can omit the `Size` from the struct completely.

The thing is that the parsed data from the TOML are passed to the custom unmarshaler as `interface{}`, but in reality it is `map[string]interface{}`, thus it does not really relies on the actual struct members at all. The code of the custom unmarshaler can stay as is and handle the potential `size` key and assign it to `MinSize`.

It would be still good to add a comment to the custom unmarshaler, that the handling of `size` is kept there for backward compatibility.

Reply:
Yeah this is great, since we won't have to export another variable (which could lead to more confusion)